packages/jest-resolve/package.json

```diff
@@ -8,15 +8,20 @@
   },
   "license": "MIT",
   "main": "build/index.js",
+  "types": "build/index.d.ts",
   "dependencies": {
+    "@jest/types": "^24.1.0",
     "browser-resolve": "^1.11.3",
     "chalk": "^2.0.1",
-    "realpath-native": "^1.0.0"
+    "realpath-native": "^1.1.0"
   },
   "devDependencies": {
     "@types/browser-resolve": "^0.0.5",
     "jest-haste-map": "^24.0.0"
   },
+  "peerDependencies": {
+    "jest-haste-map": "^24.0.0"
```

Comment:
I think this makes the most sense - the constructor of `jest-resolve` takes `ModuleMap` from `jest-haste-map`. So both for types but also for usage, we need to ensure that the API is what we expect it to be